cmake/coverage.cmake

```diff
@@ -6,6 +6,12 @@ include(${CMAKE_CURRENT_LIST_DIR}/Gcov.cmake)
 
 set(COVERAGE_ENABLED OFF BOOL)
 
+if(NOT USE_EXTERN_QUAZIP)
+  SET(LCOV_THIRDPARTY "\"*/third_party/*\"")
```

Comment:
+1, thanks :)

Reply:
Tried to do the coverage testing but I kept getting an empty folder for output to genhtml.
I wasn't sure if it was some setting I was missing in doing  the setup, especially after reading through the README (`ninja coverage`).

Reply:
`ninja coverage` works on my machine but e.g. not on TravisCI, so it may be very likely a bug on our side if it doesn't work for you. :)

Reply:
This sounds like a missing `something` in the settings to me. Some little nugget set in a user profile that makes everything 'just work' but hasn't been migrated to project configuration. But that's a guess on my part.

Reply:
Yeah, I'll investigate that issue next week after my vacation (I won't be able to read issues for the next ~1week) :)